cime_config/buildlib

```diff
@@ -116,7 +116,12 @@ def buildlib(bldroot, libroot, case, compname=None):
         mod_def_in = case.get_value("WW3_MODDEF")
         if os.path.isfile(mod_def_in):
             import filecmp
-            if not filecmp.cmp(mod_def_in, os.path.join(rundir, "mod_def.ww3")):
+            copy_file = False
+            if not os.path.isfile(os.path.join(rundir, "mod_def.ww3")): 
+                copy_file = True
+            elif not filecmp.cmp(mod_def_in, os.path.join(rundir, "mod_def.ww3")):
```

Comment:
This does a shallow compare (only compare file type, size, and modification time). To compare file contents (if necessary), you would use:
```suggestion
            elif not filecmp.cmp(mod_def_in, os.path.join(rundir, "mod_def.ww3"), shallow=False):
```